tensorrt_llm/_torch/models/modeling_speculative.py

```
@@ -1070,99 +1242,107 @@ def dflash_forward(
+                    q_for_rope = q_all
+                    k_noise_for_rope = k_noise_all
+                Q = _rope(q_for_rope.reshape(B, block_size, num_heads_per_rank,
+                                             head_dim).transpose(1, 2),
+                          q_rope_cos,
+                          q_rope_sin,
+                          unsqueeze_dim=1,
+                          is_neox=self._is_neox)
+                k_noise_rope = _rope(k_noise_for_rope.reshape(
+                    B, block_size, num_kv_heads_per_rank,
+                    head_dim).transpose(1, 2),
+                                     q_rope_cos,
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)